src/Styleguide/Components/ArtistCard.tsx

```diff
@@ -49,7 +50,7 @@ export const LargeArtistCard: SFC<Props> = props => (
   <BorderBox hover flexDirection="column" width="100%" height="254px">
     <Flex flexDirection="column" flexGrow="0" alignItems="center" pt={1}>
       {props.artist.image && (
-        <Avatar src={props.artist.image.cropped.url} mb={1} />
+        <Avatar src={get(props.artist.image, i => i.cropped.url)} mb={1} />
```

Comment:
And here the whole Avatar is rendered conditionally on image object presence, but strangely seems like image might be missing `cropped` sometimes, though not sure if it's just on staging...